app/src/test/shared/fetch/helpers.ts

```diff
@@ -137,26 +136,26 @@ export class FetchHelper<TPayload, TActionPayload> {
             });
         });
 
-        it("triggers logout when server returns bearer token error", (done: DoneCallback) => {
-            const errors: Array<ErrorInfo> = [
-                {
-                    code: "bearer-token-invalid",
-                    message: "Something wicked this way comes"
-                }
-            ];
-            this.sandbox.setStub(LoginSource, "clearShinyToken");
-            this.config.prepareForFetch();
-            this.testFetchWithMockedResponse({
-                done,
-                payload: mockResult(null, errors, "failure"),
-                errorMessage: null,
-                expectedAction: {
-                    action: "AuthActions.logOut",
-                    payload: true
-                },
-                triggerFetch: this.config.triggerFetch
-            });
-        });
+
+        // it("triggers logout when server returns bearer token error", (done: DoneCallback) => {
```

Comment:
This functionality should still exist (and be tested!) and also I think needs to be tested with respect to the new `LocalService` class - i.e. we want to make sure that the user is logged out when the service gets a `token expired` error back from the API

Reply:
As discussed, the old fetcher is now just removing the token from local storage and refreshing the page on a bearer token error, which has the effect of logging the user out. It's a temporary hack and the fetcher will be phased out. The functionality of the new `LocalService` is all under test. So we can go ahead and delete this test!

Reply:
@georgearrowwood  nice job, I think we can delete this commented code now ^